README.md

```diff
@@ -36,6 +36,8 @@ public class SampleApplication extends RosieApplication {
 }
 ```
 
+**Extend from ``RosieApplication`` is needed to be able to easily use the configuration Rosie provides you related to Dependency Injection. If you do not want to use Dependency Injection in your project, you do not need to extend from ``RosieApplication``.**
```

Comment:
Use Extend**ing** here and below, or just reword it to `Is needed to extend from... to be able to...`/`Is not mandatory to extend from...`